eng/Version.Details.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.CodeAnalysis" Version="3.3.1">
+    <Dependency Name="Microsoft.CodeAnalysis" Version="3.11.0">
       <Uri>https://github.com/dotnet/roslyn</Uri>
-      <Sha>66a912c9463eebe832cf742d2fe8bb2e1a4600ec</Sha>
+      <Sha>ae1fff344d46976624e68ae17164e0607ab68b10</Sha>
     </Dependency>
     <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build-reference-packages" Version="8.0.0-alpha.1.23305.4">
```

Comment:
@MichaelSimons I presume we would need to updated the package version here and also update the SHA below. Can you please point me to the package feed where this package is published OR point out the new package version to use?

Reply:
@MichaelSimons @mmitche Can you please help here?

Reply:
A darc subscription should be defined to automatically update this dependency.  You can use the darc cli to find the latest version info and feed.

``` bash
> darc get-asset --name Microsoft.SourceBuild.Intermediate.source-build-reference-packages
Looking up assets with name 'Microsoft.SourceBuild.Intermediate.source-build-reference-packages' in the last 30 days
Microsoft.SourceBuild.Intermediate.source-build-reference-packages @ 8.0.0-alpha.1.23328.2
Repository:    https://github.com/dotnet/source-build-reference-packages
Branch:        main
Commit:        5740876021887c3a50d3dae34cc125dbab36316f
Build Number:  20230628.2
Date Produced: 2023-06-28 11:31 AM
Build Link:    https://dev.azure.com/dnceng/internal/_build/results?buildId=2211175
BAR Build Id:  183409
Released:      False
Channels:
- .NET Eng - Latest
Locations:
- https://dev.azure.com/dnceng/internal/_apis/build/builds/2211175/artifacts (Container)
- https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json (NugetFeed)

...
```